css/main.css

```diff
@@ -325,3 +329,84 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+.contact-form {
+  background-color: white;
+  margin: auto;
+  max-width: 100vw;
+  overflow: hidden;
+}
+
+.contact-form h3 {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 22px;
+  line-height: 44px;
+  color: #172b4d;
+  margin-left: 30px;
+  margin-right: 30px;
+  padding: 0;
+  text-align: center;
+}
+
+.contact-form form {
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+  max-width: 600px;
+  margin: 0 auto;
+}
+
+.contact-form form input,
+form textarea {
+  margin-bottom: 1rem;
+  padding: 0.5rem;
+  border: 1px solid #ccc;
+}
+
+.contact-form button {
+  background-color: #f76c01;
+  color: white;
+  padding: 0.75rem 1.5rem;
+  border: none;
+  cursor: pointer;
+  margin-bottom: 1rem;
+  margin-left: 130px;
+  font-size: 1rem;
+}
+
+.contact-form button:hover {
+  color: #f76c01;
+  background-color: #fef5e1;
+  border: #f76c01;
+}
+
```

Comment:
- [ ] As the previous reviewer requested, please implemented the different button states. Currently, your button does not respond to any state changes.

***figma button states***
![figma1](https://user-images.githubusercontent.com/99656687/211118062-034cb164-99c4-4e24-bcc4-66f35defed4d.png)